spec/factory_bot/attribute_list_spec.rb

```diff
@@ -1,26 +1,35 @@
+def build_attribute_list(*attributes)
+  FactoryBot::AttributeList.new.tap do |list|
+    attributes.each { |attribute| list.define_attribute(attribute) }
+  end
+end
+
 describe FactoryBot::AttributeList, "#define_attribute" do
```

Comment:
I noticed there are separate top-level describe blocks used to test each method in the AttributeList class (as opposed to one top-level describe block and multiple inner blocks, which is what I'm used to seeing in other code bases).  Does the former choice have any benefits (or drawbacks) which don't occur in the latter?